packages/adapter-knex/lib/adapter-knex.js

```diff
@@ -680,7 +680,10 @@ class QueryBuilder {
   }
 
   _getQueryConditionByPath(listAdapter, path, tableAlias) {
-    const dbPath = path.split('_', 1);
+    const dbPath = path
+      .split('_')
+      .filter(a => a)
+      .join('_');
```

Comment:
The real goal here is to filter out `_somethingMeta` to be `somethingMeta`- not sure if this fix is too arcane and too cute.